lit/SwiftExpr/TypeLookup.test

```diff
@@ -0,0 +1,13 @@
+// Test that type lookup chooses the right language.
```

Comment:
Can we change the name of this test.  You didn't do this, but...  The point of the test is that, having imported Foundation, we can lookup a type from it.  So it's really a test of importing Foundation, type lookup is really just the means.  It would actually be better to have the test try to use Foundation...  And testing the functionality of "type lookup" which is a straight-up lldb command isn't best done in a REPL test...

Reply:
I'll find a better name. Maybe ImportFoundation.test.